pkg/machinepool/helper.go

```diff
@@ -189,7 +199,7 @@ func getMachinePoolAvailabilityZones(r *rosa.Runtime, cluster *cmv1.Cluster, mul
 	return cluster.Nodes().AvailabilityZones(), nil
 }
 
-func maxReplicaValidator(minReplicas int, multiAZMachinePool bool) interactive.Validator {
+func maxReplicaValidator(minReplicas int, replicaSizeValidationArgs *ReplicaSizeValidationArgs) interactive.Validator {
```

Comment:
Could be part of the ReplicaSizeValidationArgs struct instead

Reply:
https://github.com/openshift/rosa/pull/2662#discussion_r1870186738